sled-agent/src/rack_setup/service.rs

```diff
@@ -71,14 +68,20 @@ impl Service {
     /// - `config`: The config file, which is used to setup the rack.
     /// - `peer_monitor`: The mechanism by which the setup service discovers
     ///   bootstrap agents on nearby sleds.
-    pub fn new(
+    /// - `sled_agent_requests`: Communication channel by which we can send
+    ///    commands to local sled-agent (e.g., to initialize other sleds).
```

Comment:
When we originally chatted, I thought we discussed having this communication channel for our own sled agent - sounds like RSS is using this channel as a proxy for communicating with *all* sled agents now. Is that right?

Reply:
Yes, with the (significant) caveat that I was sloppy with the names here and this is limited to _bootstrap agent_ activities. I'll reword to clarify.

Reply:
Reworded as a part of 92aa2e2a